src/redux/reducers/rootReducer.js

```diff
@@ -0,0 +1,49 @@
+import { combineReducers } from 'redux';
+import { authReducer } from './authReducer';
+import { tripReducer } from './tripReducer';
+import { planReducer } from './planReducer';
+import { tranReducer } from './tranReducer';
+import { accomReducer } from './accomReducer';
+
+const initialState = {
+  user: {
+    lastUpdated: null,
+    isAuthenticated: false,
+    isFetching: false,
+    user: null
+  },
+  trips: {
+    lastUpdated: null,
+    isFetching: false,
+    isSynced: false,
+    trips: []
+  },
+  plans: {
+    lastUpdated: null,
+    isFetching: false,
+    isSynced: false,
+    plans: []
+  },
+  trans: {
+    lastUpdated: null,
+    isFetching: false,
+    isSynced: false,
+    trans: []
+  },
+  accoms: {
+    lastUpdated: null,
+    isFetching: false,
+    isSynced: false,
+    accoms: []
+  }
+};
```

Comment:
Is this ever used? I'm pretty sure this entire variable can be removed. However, I do like having it as a reference for where to find info in the state...

Reply:
Removed.